web/src/main/java/se/inera/intyg/intygstjanst/web/integration/message/MessageController.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2020 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.intygstjanst.web.integration.message;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.springframework.beans.factory.annotation.Autowired;
+import se.inera.intyg.infra.monitoring.annotation.PrometheusTimeMethod;
+import se.inera.intyg.intygstjanst.web.service.MessageService;
+
+/**
+ * Internal REST endpoint to retrieve messages on certificates
+ */
+@Path("/message")
+public class MessageController {
+
+    @Autowired
+    private MessageService messageService;
+
+    @PrometheusTimeMethod
+    @GET
+    @Path("/{certificateId}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    public Response findMessagesByCertificateId(@PathParam("certificateId") String certificateId) {
+        final var messageList = messageService.findMessagesByCertificateId(certificateId);
+        return Response.ok(messageList).build();
```

Comment:
@mww-sopra-steria Finns det nåt scenario där det kan bli nåt annat än HTTP 200?

Reply:
Jag skulle kunna slänga in en parametercheck så att man verkligen skickat in ett intygsid. Men i övrigt så kan jag inte se några felscenarion. Jag fixar det, committar och mergar.